der/derive/src/choice/variant.rs

```diff
@@ -153,7 +154,7 @@ mod tests {
                 ::der::Tag::ContextSpecific {
                     constructed: false,
                     number: ::der::TagNumber::N0,
-                } => Ok(Self::ImplicitVariant(decoder.decode()?.try_into()?)),
+                } => Ok(Self::ImplicitVariant(decoder.decode()?)),
```

Comment:
Can you add a test for a case where `.try_into()` should be emitted?

Reply:
Done.